px-data-grid-filter-entity.html

```diff
@@ -216,6 +218,14 @@
           if (!this.columns) {
             return;
           }
+
+          if (columnId === '-any-') {
```

Comment:
if we use this magic string, then we need to define it everywhere in documentation as reserved keyword